lib/tasks/demo.rake

```diff
@@ -31,27 +31,27 @@ namespace :demo do
     print 'Creating enroller users...'
 
     enroller1 = User.new(email: 'state1_enroller@example.com', password: '1234567ab!', jurisdiction: jurisdictions[:state1], force_password_change: false, authy_enabled: false, authy_enforced: false)
-    enroller1.add_role :enroller
+    enroller1.role = 'enroller'
```

Comment:
This doesn't just apply to these changes but also elsewhere - should we try and make the ROLES an enum in the user model so we're not dealing with hardcoded strings in places and instead can say something like `User::ROLES.ENROLLER`?